tools/sdk-cli/src/command_handlers.rs

```diff
@@ -68,549 +47,609 @@ impl Highlighter for CliHelper {
+            }
+            Commands::PrepareRedeemOnchainFunds {
+                to_address,
+                sat_per_vbyte,
+            } => {
+                let resp = self
+                    .sdk()?
+                    .prepare_redeem_onchain_funds(PrepareRedeemOnchainFundsRequest {
+                        to_address,
+                        sat_per_vbyte,
+                    })
+                    .await?;
+                serde_json::to_string_pretty(&resp).map_err(|e| e.into())
+            }
+            Commands::ListLsps {} => {
+                let lsps = self.sdk()?.list_lsps().await?;
+                serde_json::to_string_pretty(&lsps).map_err(|e| e.into())
+            }
+            Commands::LspInfo {} => {
+                let lsp_info = self.sdk()?.lsp_info().await?;
+                serde_json::to_string_pretty(&lsp_info).map_err(|e| e.into())
+            }
+            Commands::ConnectLSP { lsp_id } => {
+                self.sdk()?.connect_lsp(lsp_id).await?;
+                Ok("LSP connected successfully".to_string())
+            }
+            Commands::OpenChannelFee {
+                amount_msat,
+                expiry,
+            } => {
+                let res = self
+                    .sdk()?
+                    .open_channel_fee(breez_sdk_core::OpenChannelFeeRequest {
+                        amount_msat,
+                        expiry,
+                    })
+                    .await?;
+                serde_json::to_string_pretty(&res).map_err(|e| e.into())
+            }
+            Commands::NodeCredentials {} => match self.sdk()?.node_credentials().await? {
+                Some(credentials) => {
+                    serde_json::to_string_pretty(&credentials).map_err(|e| e.into())
+                }
+                None => Ok("No credentials".into()),
+            },
+            Commands::NodeInfo {} => {
+                serde_json::to_string_pretty(&self.sdk()?.node_info()?).map_err(|e| e.into())
+            }
+            Commands::ConfigureNode { close_to_address } => {
+                self.sdk()?
+                    .configure_node(breez_sdk_core::ConfigureNodeRequest { close_to_address })
+                    .await?;
+                Ok("Node configured successfully".to_string())
+            }
+            Commands::ListFiat {} => {
+                serde_json::to_string_pretty(&self.sdk()?.list_fiat_currencies().await?)
+                    .map_err(|e| e.into())
+            }
+            Commands::FetchFiatRates {} => {
+                serde_json::to_string_pretty(&self.sdk()?.fetch_fiat_rates().await?)
+                    .map_err(|e| e.into())
+            }
+            Commands::CloseLSPChannels {} => {
+                let tx_ids = self.sdk()?.close_lsp_channels().await?;
+                Ok(format!("Closing transaction ids:\n{tx_ids:?}"))
+            }
+            Commands::Disconnect {} => {
+                self.sdk()?.disconnect().await?;
```

Comment:
Now that it is not static anymore I think it would be nice to nullify the sdk instance allowing the user to re-connect again.